tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "declaration": true,
```

Comment:
Ah I think my previous comment about generating type files is incorrect since it is enabled here.